test/recipes/30-test_evp.t

```diff
@@ -41,6 +41,7 @@ my @files = qw(
                 evpciph_aes_ccm_cavs.txt
                 evpciph_aes_common.txt
                 evpciph_aes_cts1.txt
+                evpciph_aes_cts23.txt
```

Comment:
Is it worthwhile merging these two files together and just have `evpciph_aes_cts.txt`?

Reply:
Done